R/biometrics_article.R

```diff
@@ -0,0 +1,21 @@
+#' Biometrics Journal format.
+#'
+#' This format was adapted from the Biometrics Macro package.
+#'
+#'
+#' @inheritParams rmarkdown::pdf_document
+#' @param ... Arguments to \code{rmarkdown::pdf_document}
+#' @return R Markdown output format to pass to \code{\link[rmarkdown:render]{render}}
+#' @examples
+#' \dontrun{
+#' rmarkdown::draft("MyArticle.Rmd", template = "biometrics_article", package = "rticles")
+#' }
+#'
+#' @export
+biometrics_article <- function(..., keep_tex = TRUE, citation_package = 'none'){
```

Comment:
Are you sure you don't want to set `citation_package = 'natbib'` as the default?